flowauth/backend/flowauth/admin.py

```diff
@@ -540,7 +541,10 @@ def edit_group_servers(group_id):
                     for name, val in right["permissions"].items():
                         setattr(perm, name, val)
                         if val and not getattr(server_capability, name):
-                            raise InvalidUsage("capability_not_allowed_on_server")
+                            raise InvalidUsage(
+                                f"{name} not enabled for this server.",
```

Comment:
Very minor note: I often find it helpful to be very explicit in the phrasing of the error messages, to give an indication how to fix them. Here I suggest rephrasing this as `f"Permission (or capability) {name} not enabled for this server."

Example: "poll not enabled for this server" leaves me wondering where to look in order to enable it. (Is it something I can change in the web interface? Or is it something that needs to be configured in the server itself when flowkit is installed?)

By contrast, "Permission 'poll' not enabled for this server" makes it more obvious that I need to look in the permissions tab of the web interface. If we like we can even add a hint such as "please check the 'permissions' tab".

Anyway, I'm ok to leave it as it is but I have stumbled across a couple of these things in the past so thought I'd mention it.